File: allspice/Models/Recipe.cs
namespace allspice.Models;

public class Recipe{
   public int Id {get;set;}
   public string Title {get;set;}
   public string Instructions {get;set;}
   public string Img {get;set;}
   public string Category {get;set;}

   public bool Archived {get;set;}
   public string CreatorId {get;set;}
   public Account Creator {get;set;}
}


// type": "object",
//     "required": [
//         "id",
//         "title",
//         "instructions",
//         "img",
//         "category",
//         "creatorId"
//     ],
//     "properties": {
//         "id": {
//             "type": "number",
//         },
//         "creatorId": {
//             "type": "string"
//         },
//         "title": {
//             "type": "string"
//         },
//         "instructions": {
//             "type": "string"
//         },
//         "img": {
//             "type": "string"
//         },
//     },

File: allspice.client/src/models/Recipe.js
export default class Recipe{
   constructor(data){
      this.id = data.id;
      this.title = data.title;
      this.instructions = data.instructions;
      this.img = data.img;
      this.category = data.category;
      this.archived = data.archived;
      this.creatorId = data.creatorId;
      this.creator = data.creator;
   }
}

// public int Id {get;set;}
// public string Title {get;set;}
// public string Instructions {get;set;}
// public string Img {get;set;}
// public string Category {get;set;}

// public bool Archived {get;set;}
// public string CreatorId {get;set;}
// public Account Creator {get;set;}